.github/workflows/package.yml

```diff
@@ -0,0 +1,55 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# GitHub recommends pinning actions to a commit SHA.
+# To get a newer version, you will need to update the SHA.
+# You can also reference a tag or branch, but the action may change without warning.
+
+name: Create and publish exporter image
+
+on:
+  push:
+    branches: ['main', 'adds-templates'] 
+
+env:
+  REGISTRY: ghcr.io/equinixmetal-helm/audit-log-verifier-exporter
+  API_IMAGE_NAME: audit-log-verifier-exporter
+  Version: v0.1.0
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@40891eba8c2bcd1309b07ba8b11232f313e86779
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date -u +'%Y-%m-%dT%H:%M:%SZ')"
+
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta-api
+        uses: docker/metadata-action@517f8b0c3b2daa800eac32a9a71024c8126d46a7
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.API_IMAGE_NAME }}
+
+      - name: Build and push Docker image
+        uses: docker/build-push-action@ad44023a93711e3deb337508980b4b5e9bcdc5dc
+        with:
+          push: true
+          file: Dockerfile
+          tags: ${{ env.REGISTRY }}/${{ env.API_IMAGE_NAME }}:${{ env.Version}}
```

Comment:
Probably a better way of providing the tag you want to use? Right now each time this runs it'll just push to v0.1.0 (and so if someone isn't manually flipping this, it'll be a pain). In other repos, we've done things like push a sha tag on each push to main. And then when you cut a release or push a tag, it can use that as the version instead?

Bunch of different ways to solve that problem, so whatever makes most sense is fine. Just think as it currently stands it'll cause you some grief down the line.

Reply:
done!